### PR TITLE
Content changes for DDWRT case

### LINKED ISSUE #395: DD-WRT NXT Case Study Page - finishing touches
- [x] add screenshots and quotes as per the design: https://share.goabstract.com/6fc98f3b-b7ee-4c1f-8ed3-0677e661f087
- [x] replace breethe teaser with teaser to trainline story

### PR DESCRIPTION
**Pull request by pichfl**
- [x] Add shape once #416 is merged

closes #395

### CODE REVIEW

**Review by marcoow — CHANGES_REQUESTED** [review]
The code looks good but this reverts the content to an old state. The content on `new-simplabs` (https://gallant-hawking-42050a.netlify.com/cases/ddwrt) is actually what we want to use. Can you update?

Also this is not the client logo:

Bildschirmfoto 2019-05-20 um 14 11 29

find the correct one here: https://embedd.com

File: src/ui/components/PageCaseDdWrt/template.hbs [hunk 1]
```diff
@@ -1,91 +1,86 @@
 <div>
-  <Header @title="A modern UI for an open-source router firmware">
-    <p class="typography.lead">
-      DD-WRT is a Linux based firmware for wireless routers. Originally designed for the Linksys WRT54G series, it now runs on a wide variety of models and is installed on millions of devices worldwide.
-    </p>
-  </Header>
-  <!--container-->
+  <Header @title="a modern UI for an open-source router firmware" />
```
Comment (marcoow) [inline src/ui/components/PageCaseDdWrt/template.hbs]:
```suggestion
  <Header @title="A modern UI for an open-source router firmware" />
```

File: src/ui/components/PageCaseDdWrt/template.hbs [hunk 2]
```diff
@@ -1,91 +1,86 @@
...
-              <h3 class="typography.headline-2">
-                The Solution
-              </h3>
-              <p class="typography.body-text">
-                We completed the prototype implementation of the architecture in just over 6 weeks, defining the
-                structure for the application and establishing good Ember.js patterns. That prototype laid the
-                foundation for subsequent engineering efforts taken on by embeDD's in-house engineering team.
-              </p>
-            </div>
+            <div class="main-block.content-section-item"></div>
```
Comment (marcoow) [inline src/ui/components/PageCaseDdWrt/template.hbs]: Is this necessary?

File: src/ui/components/PageCaseDdWrt/template.hbs [hunk 3]
```diff
@@ -1,91 +1,86 @@
...
           <div class="main-block.content-section">
-            <div class="main-block.content-section-item">
-              <h3 class="typography.headline-2">
-                Enabling long term success
-              </h3>
-              <p class="typography.body-text">
-                In order to enable the client to continue the project successfully and ensure it would be a long term success, we coached embeDD's engineers and reviewed their work for an extended period of time after handing over the project.
-              </p>
-            </div>
+            <div class="main-block.content-section-item"></div>
```
Comment (marcoow) [inline src/ui/components/PageCaseDdWrt/template.hbs]: Is this necessary?

File: src/ui/components/PageCaseDdWrt/template.hbs [hunk 4]
```diff
@@ -1,91 +1,86 @@
...
+        </p>
+        <h3 class="typography.headline-2">
+          Enabling long term success
+        </h3>
+        <p class="typography.body-text">
+          In order to enable the client to continue the project successfully and ensure it would be a long term success, we coached embeDD's engineers and reviewed their work for an extended period of time after handing over the project.
+        </p>
       </div>
-      <!--sidebar-->
+      <div class="main-block.sidebar"></div>
```
Comment (marcoow) [inline src/ui/components/PageCaseDdWrt/template.hbs]: Is this necessary?

**Comment by pichfl** [comment]
@marcoow this should be ready now

File: src/ui/components/PageCaseDdWrt/template.hbs [hunk 5]
```diff
@@ -1,91 +1,86 @@
...
-            embeDD
-          </h3>
-          <p class="card.text">
-            embeDD is the company behind DD-WRT, the Linux-based open source router firmware running on millions of wireless routers around the world.
+        </Card>
+        <Card @label="Our Expertise" @title="Ember.js" @src="/assets/images/card/ember.svg">
+          <ArrowLink @href="/expertise/ember">
+            Learn More
+          </ArrowLink>
+        </Card>
```
Comment (marcoow) [inline src/ui/components/PageCaseDdWrt/template.hbs]: Let's use `<CardEmberExpertise />` here

**Review by marcoow — APPROVED** [review]

**Pull request merged**